api/v1beta1/ansibletest_types.go

```diff
@@ -128,10 +129,12 @@ type AnsibleTestWorkflowSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// +kubebuilder:validation:Optional
 	// Extra configmaps for mounting in the pod
-	ExtraMounts []extraConfigmapsMounts `json:"extraMounts,omitempty"`
+	ExtraConfigmapsMounts []extraConfigmapsMounts `json:"extraConfigmapsMounts,omitempty"`
 
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MaxLength:=100
+	// +kubebuilder:default:=""
```

Comment:
Let's keep the default value unset. I know tobiko has the default value set for stepName. Ella is working on unifying all the common parameters and this will be one of the requirements. 
```suggestion
```